com.microsoft.mrtk.uxcore/Interop/UGUIInputAdapter.cs

```diff
@@ -143,6 +122,24 @@ protected override void Awake()
                 InteractionManager.interactorRegistered += OnInteractorRegistered;
                 InteractionManager.interactorUnregistered += OnInteractorUnregistered;
             }
+
+            // Some interactors (all?) get registered before we have a chance to subscribe to the
+            // registration events. As a result; let's manually scrape the manager for the proxy interactor.
+            if (proxyInteractor == null && ThisInteractable != null && InteractionManager != null)
+            {
+                // Go find the proxy interactor that this shim is associated with.
+                interactorQueryList.Clear();
+                InteractionManager.GetRegisteredInteractors(interactorQueryList);
```

Comment:
nit-ish: Is `InteractionManager` something we ship? If so (or either way), `GetRegisteredInteractors` should really handle the clear of whatever list we pass in instead of us needing to do it manually.

Reply:
Fixed